clients/go/wire.go

```diff
@@ -0,0 +1,67 @@
+package raccoon
+
+import (
+	"errors"
+
+	"encoding/json"
+
+	"google.golang.org/protobuf/proto"
+)
+
+// WireMarshaler defines a conversion between byte sequence and raccoon request payloads.
+type WireMarshaler interface {
+
+	// Marshal marshals "any" into byte sequence.
+	Marshal(any interface{}) ([]byte, error)
+
+	// Unmarshal unmarshals "data" into "any".
+	// "any" must be a pointer value.
+	Unmarshal(data []byte, any interface{}) error
+
+	// ContentType returns the Content-Type which this marshaler is responsible for.
+	ContentType() string
```

Comment:
Do we need this to be a function? Rather could we make it a constant as this is strictly abstracting the supported media types with Raccoon.

Reply:
This is an interface, and the constant string is used in the implementation.